helpers/snakeGameGATrain.py

```diff
@@ -60,9 +61,16 @@ def __init__(self, fps, population, chroms_per_gen, bits_per_weight, num_inputs,
 
 		# Add start time of training
 		file = open("GAdata.txt", "a+")
-		file.write("******\nTraining Initialized At: " + datetime.now().strftime("%d/%m/%Y %H:%M:%S") + "\n\n")
+		file.write("******\nTraining Initialized At: " + datetime.now().strftime("%m/%d/%Y %H:%M:%S") + "\n\n")
```

Comment:
Did a little bit of [research](https://stackoverflow.com/a/17983063/11892023) and it seems like this is the best way to write the date so that it is sortable, readable, unambiguous, and easily parsable. Having Y then M then D makes sorting much much easier

```suggestion
		file.write("******\nTraining Initialized At: " + datetime.now().strftime("[%Y-%m-%d %H:%M:%S%z (%Z)]") + "\n\n")
```